homeassistant/components/switch/ecoplug.py

```diff
@@ -0,0 +1,86 @@
+"""
+EcoPlugs Component for aut-discovering Wion and EcoPlugs automatically.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.ecoplug/
+"""
+
+import logging
+
+from datetime import timedelta
+
+from homeassistant.const import (DEVICE_DEFAULT_NAME, EVENT_HOMEASSISTANT_STOP)
+from homeassistant.components.switch import SwitchDevice
+
+DEFAULT_INVERT_LOGIC = False
+
+REQUIREMENTS = ['pyecoplug==0.0.5']
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(seconds=5)
```

Comment:
Not used.

Reply:
This is a polling platform so I think it is used.